packages/styles/stories/BTP/tool-header/tool-header.stories.js

```diff
@@ -10,7 +10,7 @@ import '../../../src/tool-header.scss';
 export default {
     title: 'BTP/Tool Header',
     parameters: {
-        description: `The Tool Header exists on all BTP tools. It is the uppermost section of the tool. It is always visible to give users access to various functions.
+        description: `The Tool Header exists on all BTP tools. It is the uppermost section of the tool. It is always visible to give users access to various functions. <br><b>This component is deprecated. Please use the <a href="../?path=/docs/sap-fiori-components-shellbar--docs">SHELLBAR</a> component instead.</b>
```

Comment:
I think we agreed in the past to open the links in a new tab.

Reply:
updated. thanks!